src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs

```diff
@@ -150,6 +169,9 @@ public void Criminal_compliance_check_has_validation_error(string gatewayPageId)
             var viewModel = viewResult.Model as OrganisationCriminalCompliancePageViewModel;
             viewModel.Should().NotBeNull();
             viewModel.ErrorMessages.Count.Should().BeGreaterThan(0);
+
+            _applyApiClient.Verify(x => x.SubmitGatewayPageAnswer(It.IsAny<Guid>(), It.IsAny<string>(), It.IsAny<string>(),
```

Comment:
It.IsAny is the work of the devil, can you replace it with expected values?  Also, I don't think you need the Setup above, it should work with just the verify.